lib/postgres.js

```diff
@@ -0,0 +1,15 @@
+const { Client } = require('pg');
+
+const db = new Client({
```

Comment:
Use Pool instead of Client, should just be matter of replacing the word Client with Pool.